packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -967,6 +967,24 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             metadata,
         );
 
+        const shouldRunExpiry = this.runtimeOptions.gcOptions?.gcAllowed === true
+        && this.runtimeOptions.gcOptions?.disableGC !== true
+        && this.runtimeOptions.gcOptions?.runSweep === true;
+
+        if(shouldRunExpiry) {
+            const defaultContainerRuntimeExpiryMs = 30 * 24 * 60 * 60 * 1000;
+            const expiryMs = this.runtimeOptions.gcOptions?.testMode === true ?
```

Comment:
Instead of making this 0 for testMode, you should directly get and use the expiry value from an option.